src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/compilation_task.kt

```diff
@@ -97,17 +114,68 @@ internal fun JvmCompilationTask.produceSourceJar() {
   }
 }
 
+internal fun encodeMap(options: Map<String, String>): String {
```

Comment:
FYI - there is a CLI interface to KAPT now which handles the encoding of parameters `org.jetbrains.kotlin.kapt.cli`

Reply:
Pick that up shortly -- though it would be a breaking change for older plugin versions.